scan_beams: 1080
ttc_thre: 0.4